src/include/function/utility/function_string_bind_data.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "function/function.h"
+
+namespace kuzu {
+namespace function {
+
+struct FunctionStringBindData : public FunctionBindData {
+    std::string str;
+
+    explicit FunctionStringBindData(std::string str)
+        : FunctionBindData{common::LogicalType::STRING()}, str{str} {}
```

Comment:
str{std::move(str)}
And we usually define fields after methods. So i would move `std::string str` after constructor.